cmd/templates/status.html

```diff
@@ -128,9 +128,9 @@ <h3>Combined Instances</h3>
                         <code>{{ .EC2.State }}</code>
                     {{ end }}
                 </td>
-                <td>{{ PrettyTime .EC2.LaunchTime }}</td>
-                <td>{{ PrettyTime .ASG.TriggeredAt }}</td>
-                <td>{{ PrettyTime .EC2.TerminationTime }}</td>
+                <td>{{ PrettyTimeUTC .EC2.LaunchTime }}</td>
```

Comment:
I don't understand why the name changed.

Reply:
Because of [this](https://github.com/rebuy-de/node-drainer/blob/master/cmd/server.go#L21:L25). I wanted to use the stuff from the SDK, but the `useUTC` parameter is missing there and I wasn't able to override the default `PrettyTime` func, so I introduced the `PrettyTimeUTC` func instead.

Reply:
So we have to show UTC in the web ui, now? This flag was introduced to work around an issue with tests, but IMO the ui should still show local time.